app/scripts/modules/kubernetes/src/v2/manifest/artifact/artifact.component.ts

```diff
@@ -0,0 +1,23 @@
+import { IComponentOptions, IController, module } from 'angular';
+
+import { IArtifact } from '@spinnaker/core';
+
+class KubernetesManifestArtifactCtrl implements IController {
```

Comment:
You can omit this controller if you want, though leave it if you're planning on adding more functionality to this component.

Reply:
I was going to read some of the fields to decide to make clickable links later